ksp_plugin/plugin.cpp

```diff
@@ -152,27 +122,34 @@ void Plugin::InsertCelestialAbsoluteCartesian(
 
 void Plugin::InsertCelestialJacobiKeplerian(
     Index const celestial_index,
-    Index const parent_index,
+    std::experimental::optional<Index> const& parent_index,
     KeplerianElements<Barycentric> const& keplerian_elements,
```

Comment:
We should not have Keplerian elements for the sun, so this should be optional (and that would obviate the need for dummy 0s in the call).